Be able to dynamically configure raster base path (#10248)
camptocamp/c2cgeoportal: doc/integrator/objectstorage.rst

`Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_

.. note::

>    If you want to be able to use different Bucket or Container in different environment

Reviewer: Suggestion:

If you want to use different buckets or containers in different environments (such as integration / production)

>    (like integration / production), you should have a file names ``<name>.raster``

Reviewer: I don't understand what these file names refer to.

Maybe an example would help.

Author: I try to clarify ...

`Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_

.. note::

   If you want to use different buckets or containers in different environments

>    (such as integration / production), you should add an empty file names ``<name>.raster``

Reviewer 2: ```suggestion
   (such as integration / production), you should add an empty file named ``<tileindexbasename>.raster``
```

   (not ``<name>.shp.raster``) and a ``RASTER_BASE_PATH`` environment variable in your

>    config container, then the base path will be replaced (same number of folder).

Reviewer 2: ```suggestion
   config container, then the base path will be replaced (same number of folders).
```

* ``Object key``: <folder>/index.vrt.

You can add a vector layer in an analogous manner.

Server
~~~~~~

Fill the required environment variables.

Exoscale:

* ``AWS_ACCESS_KEY_ID``: The project access key.
* ``AWS_SECRET_ACCESS_KEY``: The project secret key.
* ``AWS_DEFAULT_REGION=ch-dk-2``: Should already be in your env.project.
* ``AWS_S3_ENDPOINT=sos-ch-dk-2.exo.io``: Should already be in your env.project.

Azure docker-compose:

* ``AZURE_STORAGE_CONNECTION_STRING``: The connection string.

For Azure AKS the access should be given by the AzureAssignedIdentity in Kubernetes,

.. note::

   If you want to use different buckets or containers in different environments
   (such as integration / production), you should add an empty file names

>    ``<name>.ggs.raster`` or ``<name>.ggz.raster``

Reviewer 2: ```suggestion
   ``<name>.qgs.raster`` or ``<name>.qgz.raster``
```
